diff -urN xgraph/xgraph.h xgraph.patch/xgraph.h
--- xgraph/xgraph.h	Tue Oct 12 17:53:50 1993
+++ xgraph.patch/xgraph.h	Fri Mar 22 12:11:07 2002
@@ -146,13 +146,11 @@
 extern char *Realloc();
 #ifdef _AIX370
 extern int sprintf();
-#else
-extern char *sprintf();
 #endif
 extern char *strcpy();
 extern char *strcat();
-extern char *rindex();
-extern char *index();
+/*extern char *rindex();
+extern char *index();*/
 extern void exit();
 extern void Free();
 extern double atof();
