diff -urN xgraph/Makefile xgraph.patch/Makefile
--- xgraph/Makefile	Tue Sep 14 04:17:53 1993
+++ xgraph.patch/Makefile	Fri Mar 22 12:16:34 2002
@@ -93,7 +93,7 @@
            CFLAGS = $(CDEBUGFLAGS) $(CCOPTIONS) $(ALLDEFINES)
         LINTFLAGS = $(LINTOPTS) -DLINT $(ALLDEFINES)
            LDLIBS = $(SYS_LIBRARIES) $(EXTRA_LIBRARIES)
-        LDOPTIONS = $(CDEBUGFLAGS) $(CCOPTIONS)
+        LDOPTIONS = $(CDEBUGFLAGS) $(CCOPTIONS) -L$(LIBDIR)
    LDCOMBINEFLAGS = -X -r
 
         MACROFILE = sun.cf
@@ -117,15 +117,15 @@
 # $XConsortium: Project.tmpl,v 1.63 89/12/18 16:46:44 jim Exp $
 
           PATHSEP = /
-        USRLIBDIR = $(DESTDIR)/usr/lib
-           BINDIR = $(DESTDIR)/usr/bin/X11
-          INCROOT = $(DESTDIR)/usr/include
+        USRLIBDIR = $(DESTDIR)/usr/X11R6/lib
+           BINDIR = $(DESTDIR)/usr/X11R6/bin
+          INCROOT = $(DESTDIR)/usr/X11R6/include
      BUILDINCROOT = $(TOP)
       BUILDINCDIR = $(BUILDINCROOT)/X11
       BUILDINCTOP = ..
            INCDIR = $(INCROOT)/X11
            ADMDIR = $(DESTDIR)/usr/adm
-           LIBDIR = $(USRLIBDIR)/X11
+           LIBDIR = $(USRLIBDIR)
         CONFIGDIR = $(LIBDIR)/config
        LINTLIBDIR = $(USRLIBDIR)/lint
 
@@ -185,7 +185,7 @@
      MKFONTDIRSRC = $(FONTSRC)/mkfontdir
      EXTENSIONSRC = $(TOP)/extensions
 
-  DEPEXTENSIONLIB = $(USRLIBDIR)/libXext.a
+  DEPEXTENSIONLIB = $(USRLIBDIR)/libXext.so
      EXTENSIONLIB =  -lXext
 
           DEPXLIB = $(DEPEXTENSIONLIB)
